engine/engine.go

```diff
@@ -122,13 +123,27 @@ func (e *Engine) Run(ctx context.Context, step *spec.Step, output io.Writer) (*r
 		vol.Path = pathConverter(vol.Path)
 	}
 
+	printCommand(step, output, isDrone)
```

Comment:
i would refactor this a little, as it doesnt follow the look and feel in the rest of this function

ie 
```
if !isDrone && len(step.Command) > 0 {
   printCommand(step, output)
end
```
then remove the conditional from the function